google/cloud/bigquery/job.py

```diff
@@ -3247,7 +3247,7 @@ def to_arrow(
                 ``'tqdm_gui'``
                   Use the :func:`tqdm.tqdm_gui` function to display a
                   progress bar as a graphical dialog box.
-            bqstorage_client (google.cloud.bigquery_storage_v1beta1.BigQueryStorageClient):
+            bqstorage_client (google.cloud.bigquery_storage_v1.BigQueryReadClient):
                 **Beta Feature** Optional. A BigQuery Storage API client. If
```

Comment:
nit: Still beta with this change?

Reply:
Shouldn't be, since BQ Storage is now stable. Forgot to remove the `beta` mark here.